tripy/examples/resent50/resnet50.ipynb

```diff
@@ -0,0 +1,529 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# ResNet50 Image Classificiation with `tripy` - An Implementation and Demo\n",
```

Comment:
I think we can shorten the title:
```suggestion
    "# ResNet50 Image Classificiation",
```
`tripy` and "Demo" are implied by the location of the notebook